packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TWalletInfo } from 'Types';
+import { Text, WalletCard } from '@deriv/components';
+import { useCurrencyConfig } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { getWalletCurrencyIcon } from '@deriv/utils';
+import wallet_description_mapper from 'Constants/wallet_description_mapper';
+
+type TAddWalletCard = {
+    wallet_info: React.PropsWithChildren<TWalletInfo>;
+};
+
+const AddWalletCard = observer(({ wallet_info }: TAddWalletCard) => {
+    const {
+        ui: { is_dark_mode_on, is_mobile },
+    } = useStore();
+
+    const { currency = '', landing_company_name, is_added, gradient_card_class } = wallet_info;
+    const { getConfig } = useCurrencyConfig();
+    const currency_config = getConfig(currency);
+
+    const wallet_details = {
+        currency,
+        icon: getWalletCurrencyIcon(currency, is_dark_mode_on),
+        icon_type: currency_config?.type,
+        jurisdiction_title: landing_company_name?.toUpperCase(),
```

Comment:
clarification: will this work for Maltainvest? 
Shouldnt the jurisdiction_title be 'MALTA' for maltainvest?

Reply:
hey @amina-deriv 
i checked you concern and found out that we modify landing_company_name before we use it here in `useWalletsList` hook:
```
/** Landing company shortcode the account belongs to. */
landing_company_name: wallet.landing_company_name?.replace('maltainvest', 'malta'),
```